.github/workflows/deploy_website.yaml

```
@@ -9,12 +9,11 @@ on:
   workflow_dispatch:
     inputs:
       environment:
-        description: 'The environment used as target'
+        description: 'The tenant environment used as target'
```

```suggestion
        description: 'The Tenant for which the static site will be built
```
I would keep the description consistent.